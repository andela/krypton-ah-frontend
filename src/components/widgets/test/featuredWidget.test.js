import React from 'react';
import { shallow } from 'enzyme';
import Widget from '../featuredwidget';

describe('button test', () => {
  it('should test for the button component', () => {
    const descriptionProps = 'this is a test description';
    const classNameProps = 'className';
    const wrapper = shallow(<Widget description={descriptionProps} className={classNameProps} />);
    expect(wrapper.find('Feed')).toBeDefined();
    expect(wrapper.find('featuredwidget')).toBeDefined();
  });
});
