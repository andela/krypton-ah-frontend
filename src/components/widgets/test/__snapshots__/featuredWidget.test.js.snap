// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`widget test should test for the Icons component 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`widget test should test for the Icons component 2`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`widget test should test for the advert component 1`] = `
<GridColumn
  className="Advert"
>
  <Image
    as="img"
    className="AdvertImg"
    src="https://res.cloudinary.com/ah-krypton/image/upload/v1548328425/mm17kbfqf7sjrw2w45rq.png"
    ui={true}
  />
</GridColumn>
`;

exports[`widget test should test for the article feed  component 1`] = `<componentloader />`;

exports[`widget test should test for the button component 1`] = `
<Card
  className="widgetIcons"
>
  <CardContent
    className={true}
  >
    <Feed>
      <MainFeaturedArticle
        date="2019-01-23T07:09:10.447Z"
        description="Descriptive title"
        id="65719288-0395-445e-b587-2b98b70bdec9"
        image="https://farm4.staticflickr.com/3894/15008518202_c265dfa55f_h.jpg"
        key="65719288-0395-445e-b587-2b98b70bdec9"
        slug="this-is-a-post-title-l78hgybf"
        title="This is a post title"
      />
    </Feed>
  </CardContent>
</Card>
`;

exports[`widget test should test for the button component 2`] = `
<Card
  className="widgetIcons"
>
  <CardContent
    className={true}
  >
    <Feed>
      <MainFeaturedArticle
        date="2019-01-23T07:09:10.447Z"
        description="Descriptive title"
        id="65719288-0395-445e-b587-2b98b70bdec9"
        image="https://farm4.staticflickr.com/3894/15008518202_c265dfa55f_h.jpg"
        key="65719288-0395-445e-b587-2b98b70bdec9"
        slug="this-is-a-post-title-l78hgybf"
        title="This is a post title"
      />
    </Feed>
  </CardContent>
</Card>
`;
